bin/ember-template-lint.js

```diff
@@ -273,7 +278,10 @@ async function run() {
   }
 
   let results = processResults(resultsAccumulator);
-  if (results.errorCount > 0) {
+
+  if (options['max-warnings'] !== -1) {
+    process.exitCode = results.errorCount + results.warningCount >= options['max-warnings'] ? 1 : 0;
```

Comment:
This behavior is a tad odd to me, and doesn't seem to match what eslint does.

For example (unless I'm misreading the diff here), if you invoke like `ember-template-lint --max-warnings=10 .` and you have 2 **errors** the command will still exit with an exit code of zero.

I think we need to check for `results.errorCount > 0 || results.maxWarnings > options.maxWarnings`, or something like that.

Reply:
I had the intent of handling both error and warnings under the same flag, but that seems odd now. Have handled only the warnings and removed the default value.